recipe/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
```

Comment:
Sorry also these hidden files should be removed to please.

Reply:
Just removed those as well